torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -315,6 +315,11 @@ void IrPrinter::handle(const BroadcastOp* bop) {
   os_ << bop->out() << " = broadcast( " << bop->in() << " )\n";
 }
 
+void IrPrinter::handle(const TransposeOp* top) {
+  indent();
+  os_ << top->out() << " = transpose( " << top->in() << " )\n";
```

Comment:
shouldn't we print the old_to_new mapping here?

Reply:
Both the input and output tensors are printed, so that's not strictly necessary.

Reply:
what is the purpose of TransposeOp::old2new_ then?

Reply:
`old2new_` has the truth. Printing out the input and output tensors is usually just fine, but sometimes it could be ambiguous. For example:

```
t0 = makeConcreteTensor({10, 10});
t1 = transpose(t0, {{0, 1}});
```

Here, both `t0` and `t1` would have `[10, 10]`, so just printing them doesn't tell how they are transposed, so we need to consult `old2new_`. However, when they are something like:

```
t0 = makeSymbolicTensor(2);
t1 = transpose(t0, {{0, 1}});
```

Then, this would look like:

```
%kernel_math {
T1[ iS2{i3}, iS3{i1} ] = transpose( T0[ iS0{i1}, iS1{i3} ] )
}
```

So, I think most of the time it should be just sufficient to print the input and output tensors.